### DISCUSSION TITLE
Hupertoken cuz

### DISCUSSION THREAD

**Discussion by Jameyrose** [community/community#86666] [Other Feature Feedback, Questions, & Ideas]
i need help getting access to an app i made called hypertoken xyz
Someone stole the app and is marketing on it with my over 10k subscribers

**Comment by ibra-kdbra** [comment]
### It’s important to take immediate action to try and recover your stolen app.

**First** contact the platform where your app was hosted. They may be able to assist you in recovering your app or at least taking it down to prevent further unauthorized use.
Also **Legal Action** is an important action to take, then secure Your Accounts.
I think you should consider making a public announcement to your subscribers about the situation. This can prevent them from being misled by the unauthorized marketing.
